src/components/ConversationItem/index.js

```diff
@@ -24,6 +25,11 @@ const ConversationItem = ({ conversation, onSelect }) => {
         </View>
       </View>
       <Image source={{ uri: topicIcon }} style={styles.topicIcon} />
+      {unreadMessages > 0 && (
+        <View style={styles.unread}>
+          <Text style={styles.unreadNumber}>{unreadMessages}</Text>
+        </View>
```

Comment:
WDYT about making this a separate component, like Badge? I think you will also use it in Main Screen, in headers right chat icon, when you have new messages. It could receive styling props to set it where you want in each case and with the colors you need.  I actually used an existing `Badge` element from `'react-native-elements'` but it is great you implemented your own! 💪 

![image](https://user-images.githubusercontent.com/8755889/81230454-e6b35e00-8fc7-11ea-98af-e8b865c9c9bc.png)

Reply:
I like this suggestion from @ManuViola77  to create a separate `Badge` component!